nncf/quantization/algorithms/weight_compression/backend.py

```diff
@@ -101,3 +106,40 @@ def do_compression(
             precision. By default, the backup precision is assigned for the embeddings and last layers.
         :return: A resulting model with compressed weights.
         """
+
+    @staticmethod
+    @abstractmethod
+    def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> TargetPoint:
+        """
+        Returns backend-specific target point.
+
+        :param target_type: Type of the location that should be modified.
+        :param target_node_name: Name of the located node.
+        :param port_id: id of the port for the statistics disctribution.
+        :return: Backend-specific TargetPoint.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def raw_statistic_collector(inplace: bool, num_samples: int = None) -> TensorStatisticCollectorBase:
+        """
+        Returns backend-specific raw statistic collector.
+        This statistic collector uses for raw data calculation, without aggregating.
+
+        :param inplace: Whether to calculate statistic inplace or not.
```

Comment:
Honestly, inplace and not inplace  raw statistic collector are the same, we can just pass False everywhere and remove this parameter from this function